### ISSUE TITLE
Berechtigungen für GET /users/{id}/performance_report

### ISSUE THREAD

**Issue by lecodeski** [hundertzehn/mocoapp-api-docs#204]
Ich hatte kürzlich den Soll-Ist in der API gesucht und bin über eine [Diskussion](https://github.com/hundertzehn/mocoapp-api-docs/issues/152) gestolpert. Der [Endpunkt](https://github.com/hundertzehn/mocoapp-api-docs/blob/master/sections/users.md#get-usersidperformance_report) wäre auch für mich genau das was ich suche.
Leider bekomme ich immer einen `403` zurück – obwohl ich in der Web-UI mein eigenen Soll-Ist natürlich einsehen kann. Auch alles andere wie Activity erstellen oder löschen geht mit dem gleichen Api-Key und User-ID.

Kann das sein, dass das Rechte-mäßig wo anders aufgehangen ist? Z. B., dass `/users` generell für „durchschnittliche“ Mitarbeiter nicht zugänglich ist? Oder ist das etwas, was wir in der Company einstellen müssen?

**Comment by agileapplications** [comment]
@lecodeski Gut hast du nochmal nachgehakt. Es ist so, `/users` ist nur mit Personalrechten abrufbar. Der eigene Soll-Ist ist so nicht abrufbar. Baut ihr für jeden MA eine eigene Übersicht? Wäre gut, hättest du noch etwas mehr Background-Info. Danke, Tobias

**Comment by lecodeski** [comment]
Ja, wir haben ein [CLI-Tool](https://github.com/lecodeski/moco_cli) programmiert und ich hatte eben noch die Überstundenfunktion eingebaut. Im Moment ist es so gelöst, dass noch ein extra Token mit Personalrechten abgefragt wird. Hätten das gerne aber sauber so gelöst, dass es mit dem jeweiligen Mitarbeiter-Token abrufbar ist, so wie die restlichen Funktionen (v. a. die Activity-Endpunkte).

**Comment by thomas-wtfoxtrot** [comment]
Hey @agileapplications ich wollte gerade ein neues Issue erstellen, weil ich genau den selben Usecase habe: ich würde gerne auf meinen eigenen performance_report zugreifen, um mir schnell und unkompliziert den Soll/Ist-Stand ausgeben zu lassen. Dass hier ein 403 zurückkommt, fühlt sich nicht korrekt an, da alle Daten daraus ja auch im Frontend für mich zugreifbar sind

**Comment by Schaukelking** [comment]
Hey @lecodeski @thomas-wtfoxtrot,
vielen Dank für eure Geduld. Seit heute gibt es nun den /profile/performance_report Endpunkt, der euer Anliegen lösen sollte!
https://api.mocoapp.com/api/docs/v1#tag/profile/GET/profile/performance_report
Dort kriegt ihr die gleiche Info wie über /users/{id}/perfromance_report - nur für den eingeloggten User und ohne Personalrechte dafür zu brauchen.

Wir räumen grade das Issue Board auf, da wir es deprecaten wollen. Für zukünftige Feature Requests bitte direkt an den Support wenden. Dort können wir Anfragen besser aggregieren und evaluieren. 

Einen schönen Start in die Woche
Patrick
